yarn.lock

```diff
@@ -3241,6 +3241,11 @@ ansi-regex@^2.0.0:
   resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-2.1.1.tgz#c3b33ab5ee360d86e0e628f0468ae7ef27d654df"
   integrity sha1-w7M6te42DYbg5ijwRorn7yfWVN8=
 
+ansi-regex@^3.0.0:
```

Comment:
yarn why ?

Reply:
I think this came when i merged. Ill check

Reply:
Yep the same happened to me

Reply:
@FredericEspiau  Any idea how to remove this ? Should i just to `yarn remove` ?